pkg/controllers/user/monitoring/clusterHandler.go

```diff
@@ -362,10 +363,15 @@ func (ch *clusterHandler) deployApp(appName, appTargetNamespace string, appProje
 		}
 	}
 
+	creator, err := ch.app.systemAccountManager.GetSystemUser(ch.clusterName)
+	if err != nil {
+		return nil, err
+	}
+
 	appCatalogID := settings.SystemMonitoringCatalogID.Get()
 	app := &v3.App{
 		ObjectMeta: metav1.ObjectMeta{
-			Annotations: monitoring.CopyCreatorID(nil, cluster.Annotations),
+			Annotations: map[string]string{creatorIDAnno: creator.Name},
```

Comment:
Same here. Do we have to copy annotations from `obj`?

Reply:
ditto